**Cite the benchmark:**
Ostroverkhova, D., Sheng, Y., & Panchenko, A. (2024). Are next-generation pathogenicity predictors applicable to cancer? Journal of Molecular Biology, 168, 168644. https://doi.org/10.1016/j.jmb.2024.168644

The comprehensive experimental benchmark of cancer driver and neutral mutations. Mutation annotations were obtained from the previous seven experimental studies:

1) Martelotto, L.G. et al. Benchmarking mutation effect prediction algorithms using functionally validated cancer-related missense mutations. Genome Biol 15, 484 (2014).

2) Olivier, M. et al. The IARC TP53 database: new online mutation analysis and recommendations to users. Hum Mutat 19, 607-14 (2002)

3) Starita, L.M. et al. Massively Parallel Functional Analysis of BRCA1 RING Domain Variants. Genetics 200, 413-22 (2015)

4) Mahmood, K. et al. Variant effect prediction tools assessed using independent, functional assay-based datasets: implications for discovery and diagnostics. Hum Genomics 11, 10 (2017)

5) Campbell, B.B. et al. Comprehensive Analysis of Hypermutation in Human Cancer. Cell 171, 1042-1056 e10 (2017)

6) Ng, P.K. et al. Systematic Functional Annotation of Somatic Mutations in Cancer. Cancer Cell 33(3), 450-462(2018)

7) Kim, E. et al. Systematic Functional Interrogation of Rare Cancer Variants Identifies Oncogenic Alleles. Cancer Discov. 6, 714â€“26(2016)

'Driver' and 'passenger' labels were assigned according to Brown AL, Li M, Goncearenco A, Panchenko AR Finding driver mutations in cancer: Elucidating the role of background mutational processes.
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006981

Check out our paper at: https://www.sciencedirect.com/science/article/pii/S0022283624002390

This supplementary table (STable_1.xlsx) includes mutations in the benchmark that are observed in the TCGA pan-cancer dataset.
